### PR TITLE
update the OpenBSD build guide

### PR DESCRIPTION
**Pull request by janstary**
This updates the instructions to OpenBSD-current
and slightly tweaks the db48 helper script.

### CODE REVIEW

**Comment by janstary** [comment]
Tavis says this fails the `multiwallet.py` and `p2p-versionbits-warning.py`.
I don't understand where I even touch that.

File: contrib/install_db4.sh [hunk 1]
```diff
@@ -1,26 +1,21 @@
...
-fi
+DBVERS="db-4.8.30.NC"
+DBFILE="$DBVERS.tar.gz"
+DBPAGE="http://download.oracle.com/berkeley-db/$DBFILE"
+DBHASH='12edc0df75bf9abd7f82f821795bcee50f42cb2e5f76a6a281b85732798364ef'
 
-expand_path() {
-  echo "$(cd "${1}" && pwd -P)"
+err() {
+	echo $@ >&2
```
Comment (gnumarcelo) [inline contrib/install_db4.sh]: As a code convention to the rest of this file I think you must remove tabs and keep two spaces indentation here.

**Review by gnumarcelo — COMMENTED** [review]
Well done!

**Comment by laanwj** [comment]
Needs rebase if still relevant (and see also #12041).

**Comment by laanwj** [comment]
Closing for now, as it's not in mergeable state. Let me know if you start work on this again, then I'll reopen.

**Comment by janstary** [comment]
I will prepare a better PR based on 0.16.0

**Pull request closed without merging**